### PR TITLE
Add test plan for managed k8s clusters (EKS, GKE, AKS)

### PR DESCRIPTION
**Pull request by yangchiu**
Add test plan for managed k8s clusters (EKS, GKE, AKS)

For [#3983](https://github.com/longhorn/longhorn/issues/3983)

Signed-off-by: Yang Chiu <yang.chiu@suse.com>

### CODE REVIEW

**Review by kaxing — APPROVED** [review]
Looks good to me. Thank you Yang.

Do we want to add suggested node size and node number in `managed-kubernetes-clusters/_index.md`?

**Comment by yangchiu** [comment]
> Looks good to me. Thank you Yang.
> 
> Do we want to add suggested node size and node number in `managed-kubernetes-clusters/_index.md`?

I updated this PR to specify the node size (3 nodes). Thank you!

But for the volume size, I don't have suggestion on it. To test expanding volume, I just start from the default size, and set the volume size a arbitrary number larger than the default size.

File: docs/content/manual/pre-release/managed-kubernetes-clusters/AKS/expand-volume.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      --kubernetes-version ${AKS_K8S_VERSION} \
+      --mode System
+    ```
+5. Using Longhorn UI to disable the disk scheduling and request eviction for nodes in the old node-pool.
+6. Cordon and drain Kubernetes nodes in the old node-pool.
+    ```
+    AKS_NODEPOOL_NAME_OLD=<old-nodepool-name>
+    for n in `kubectl get nodes | grep ${AKS_NODEPOOL_NAME_OLD}- | awk '{print $1}'`; do
+      kubectl cordon $n && \
+      kubectl drain $n --ignore-daemonsets --delete-emptydir-data
```
Comment (innobead) [inline docs/content/manual/pre-release/managed-kubernetes-clusters/AKS/expand-volume.md:31]: For draining the node, suggest excluding CSI attacher/provisioner pods to ensure volume operations correctly between longhorn CSI node plugin and kubelet. The same change to other managed K8s distros. 

ref: https://longhorn.io/docs/1.2.4/volumes-and-nodes/maintenance/#updating-the-node-os-or-container-runtime

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, just one request needed to resolve first.

**Review by kaxing — APPROVED** [review]

**Review by innobead — APPROVED** [review]

**Pull request merged**